flexmeasures/ui/templates/base.html

```diff
@@ -577,6 +577,10 @@
                 }
             {% endif %}
 
+            {% if active_subpage == "asset_graph" %}
+                getAssetKPIs();
```

Comment:
What bothered me about the UX was that it takes some time for this to be done. There are two issues, actually:

- Why is the call to /kpis happening seconds after the calls to the graphs(data) are placed? I believe it is because you placed this code within `picker.on('selected', (startDate, endDate) => {}`. My other comment below suggests moving this into the graphs template anyway, so take care of this loading time issue then.
- I feel we should use the trick that modern websites have and place two KPI dummies here (IF the asset has  `sensors_to_show_as_kpis` set, which the basic asset view can tell this template), which are replaced when the actual KPIs have finished loading.